test/form/datepicker/datepicker.directive.spec.js

```diff
@@ -0,0 +1,46 @@
+describe('Directive: pfDatepicker', function() {
+  var $scope, $compile, $timeout, element, dateInput;
+
+  beforeEach(module('patternfly.form', 'form/datepicker/datepicker.html'));
+
+  beforeEach(inject(function(_$compile_, _$rootScope_) {
+    $compile = _$compile_;
+    $scope = _$rootScope_;
+  }));
+
+  var compileDatepicker = function(markup, $scope) {
+    var el = $compile(markup)($scope);
+    $scope.$apply();
```

Comment:
I've usually seen `$scope.$digest()` used in directive testing instead of `$scope.$apply()`.  Any reason for using `$apply()`?